libs/core/src/lib/select/option/option.component.html

```diff
@@ -1,9 +1 @@
-<ng-template #contentTemplateRef><ng-content></ng-content></ng-template>
-
-<span *ngIf="!_extendedTemplate" fd-list-title>
-    <ng-container *ngTemplateOutlet="contentTemplateRef"></ng-container>
-</span>
-
-<ng-container *ngIf="_extendedTemplate">
-    <ng-container *ngTemplateOutlet="contentTemplateRef"></ng-container>
-</ng-container>
+<ng-content></ng-content>
```

Comment:
Why are we removing this part? Are we removing customization item level with custom template

Reply:
Yes Frank, as Custom template provides more flexibility.